src/containers/TopNav/TopNavDesktopItems/index.tsx

```diff
@@ -153,6 +153,9 @@ const TopNavDesktopItems = () => {
         popoverId="community-popover"
         setAnchorEl={setCommunityAnchorEl}
       />
+      <Link className="TopNavDesktopItems__right-item TopNavDesktopItems__link" tabIndex={-1} to="/developer">
+        Developer
+      </Link>
```

Comment:
Lets remove this, as we don't want this page to be linkable

Reply:
done